src/components/molecules/FilterMenu/MenuFilter.styled.tsx

```diff
@@ -0,0 +1,69 @@
+import {Checkbox, Dropdown, Menu} from 'antd';
+
+import styled from 'styled-components';
+
+import Colors from '@styles/Colors';
+import Fonts from '@styles/Fonts';
+
+export const StyledFilterMenu = styled(Menu)`
+  border: 1px solid ${Colors.greyBorder};
+
+  background-color: ${Colors.grey1000};
+`;
+
+export const StyledFilterCheckbox = styled(Checkbox)`
+  .ant-checkbox-inner {
+    background-color: ${Colors.grey1000};
+    border-color: ${Colors.greyBorder};
+  }
+  .ant-checkbox-checked .ant-checkbox-inner {
+    background-color: ${Colors.purple};
+    border: 0;
+  }
+  .ant-checkbox-input:focus + .ant-checkbox-inner {
+    border-color: ${Colors.purple};
+  }
+  color: ${Colors.grey450};
+`;
+
+export const StyledFilterMenuItem = styled(Menu.Item)`
+  &:hover,
+  &:focus {
+    background-color: ${Colors.dashboardTableBackground};
+    .ant-checkbox-inner {
+      border-color: ${Colors.purple};
+    }
+  }
+`;
+
+export const StyledFilterDropdown = styled(Dropdown)`
+  padding: 0 4px;
+
+  ${({visible}) => visible && `background-color: ${Colors.greyHover};`}
```

Comment:
1) Rename visible => isVisible
2) Use props => props.isVisible
3) Use ternary, not &&

Reply:
fixed